### PR TITLE
TV4Play.se

### PR DESCRIPTION
**Pull request by numismatika**
- add img cdn subdomains
- re-enable the rule

### CODE REVIEW

**Comment by davidhedlund** [comment]
@numismatika This bug have not been fixed in HTTPS Everywhere 5.1.10.

**Comment by numismatika** [comment]
@davidhedlund  as you can see the Pullrequest hasn't been merged yet, so not much of a surprise there.

**Comment by davidhedlund** [comment]
@numismatika Ok. Thank you for the explanation.

**Comment by J0WI** [comment]
let me bring the test result up to date...

File: src/chrome/content/rules/TV4Play.se.xml [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
+	- secure.tv4.se
 -->
-<ruleset name="TV4play.se" default_off='failed ruleset test'>
+	<target host="account.services.tv4play.se" />
+	<target host="android.tv4play.se" />
+	<target host="ios-conf.tv4play.se" />
+	<target host="ios.tv4play.se" />
+	<target host="papi.tv4play.se" />
+	<target host="prima.tv4play.se" />
+	<target host="sslapi.tv4play.se" />
```
Comment (J0WI) [inline src/chrome/content/rules/TV4Play.se.xml]: api.tv4play.se could also be redirected to sslapi.tv4play.se

File: src/chrome/content/rules/TV4Play.se.xml [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
+	<target host="www.tv4.se" />
 	<target host="www.tv4play.se" />
-	<rule from="^http://www\.tv4play\.se/" to="https://www.tv4play.se/"/>
+
+	<target host="img0.tv4cdn.se" />
+	<target host="img1.tv4cdn.se" />
+	<target host="img2.tv4cdn.se" />
+	<target host="img3.tv4cdn.se" />
+
 	<rule from="^http://tv4play\.se/" to="https://www.tv4play.se/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/TV4Play.se.xml]: Please document why this is necessary.

**Comment by J0WI** [comment]
Please move all comments above the ruleset tag.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**